### DISCUSSION TITLE
timestep in config.ini

### DISCUSSION THREAD

**Discussion by biglimp** [jixuan-chen/target#7] [General]
I have a question on timestep. How is it related to the actual meteorological forcing file? If the forcing file is in hourly resolution, can timestep be set to anything else than 60 minutes?

**Comment by jixuan-chen** [comment]
Hi Fredrik,

The model translates the dates and timestep in the config file into
nt(number of timesteps) and creates the output matrix based on that. The
model however loops through all the entries in the forcing file between the
spinup(date1a) and end(date2) without considering the actual spacing of the
data.

So theoretically you can set the timestep to anything smaller than the
actual timestep, you’ll get all the results but they’re labelled with the
wrong time (date1a+timestep*i), and half of the output matrix is unused.
Setting the timestep to larger than the actual will however lead to an
index error because the output matrix is not big enough.

**Reply by biglimp** [reply]
Ok, thanks. It sounds best than to set the timestep so that it correspond to the forcing file and not let the user in UMEP decide this.
